### DISCUSSION TITLE
电脑网络显示无网络点击进去提示只能访问内部网络，但是实际可以访问外网，求大神解惑

### DISCUSSION THREAD

**Discussion by z965489136** [vernesong/OpenClash#1827] [Q&A]
提示断网状态，ping不同百度，也影响一些软件
![photo_2021-12-07_19-11-13](https://user-images.githubusercontent.com/89550768/145019442-7131b33d-d99d-402b-8a8c-751397422ece.jpg)

**Comment by Mosney** [comment]
https://github.com/vernesong/OpenClash/issues/1839#issuecomment-993653330
